src/components/screens/transactionDetails/transactionDetails.css

```diff
@@ -107,21 +123,38 @@
       "signedAndRemainingMembersList signedAndRemainingMembersList";
   }
 
+  & .unlockToken {
+    grid-template-areas:
+      "illustration illustration"
+      "sender sender"
+      "amount amount"
+      "transactionId date"
+      "blockId blockHeight"
+      "fee confirmations"
+      "signedAndRemainingMembersList signedAndRemainingMembersList";
```

Comment:
Is the signedAndRemainingMembersList needed for the unlockToken transaction?

Reply:
no this was a mistake. I removed it.